custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -0,0 +1,99 @@
+# Needed libraries
```

Comment:
In order not to duplicate code in score.R and fit.R
I suggest the following.
1. Create a folder `r_common_code`, (need to add `__init__.py` for the sake of setup). 
2. put all the code from `score.R` and `fit.R` into `common.R`
3. and import would look like
```
CUR_DIR = os.path.dirname(os.path.abspath(__file__))
R_COMMON_PATH = os.path.abspath(os.path.join(CUR_DIR, "..", "r_common_code", "common.R"))
```